scss/partials/_hero.scss

```diff
@@ -353,13 +341,17 @@
 
     &__images {
       @include grid();
+
       grid-template-columns: repeat(12, 1fr);
       width: 100%;
-      height: 20rem;
+      height: 22vw;
```

Comment:
This tweak just resulting in better scaling of the image overlap with the hero background.